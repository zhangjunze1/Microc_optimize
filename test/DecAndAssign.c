void main() {
    int i=1;
    char a='a';
    print("%d",i);
    print("%c",a);
}

