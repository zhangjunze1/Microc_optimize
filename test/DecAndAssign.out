24 19 0 5 25 13 0 1 12 13 0 1 1 0 97 12 13 11 22 15 -1 13 0 1 1 11 23 21 2